flight/Modules/Actuator/actuator.c

```diff
@@ -691,7 +589,7 @@ static bool set_channel(uint8_t mixer_channel, float value, const ActuatorSettin
                             }
 			}
 			PIOS_Servo_Set(mixer_channel,
-							buzzOn?actuatorSettings->ChannelMax[mixer_channel]:actuatorSettings->ChannelMin[mixer_channel]);
+							buzzOn?actuatorSettings.ChannelMax[mixer_channel]:actuatorSettings.ChannelMin[mixer_channel]);
```

Comment:
Note things in this area have pre-existing mixed use of spaces / tabs, etc, are completely broken for style for other reasons.